frontends/ol-components/src/components/LearningResourceExpanded/LearningResourceExpanded.tsx

```diff
@@ -0,0 +1,425 @@
+  color: ${theme.custom.colors.silverGrayDark};
+
+  svg {
+    width: 20px;
+    height: 20px;
+    flex-shrink: 0;
+  }
+`
+
+const InfoLabel = styled.div`
+  width: 85px;
+  flex-shrink: 0;
+`
+
+const InfoValue = styled.div`
+  ${{ ...theme.typography.body2 }}
+  color: ${theme.custom.colors.black};
+`
+
+type LearningResourceExpandedProps = {
+  resource?: LearningResource
+  imgConfig: EmbedlyConfig
+}
+
+const ImageDisplay: React.FC<{
+  resource?: LearningResource
+  config: EmbedlyConfig
+}> = ({ resource, config }) => {
+  if (resource?.resource_type === "video" && resource?.url) {
+    return (
+      <EmbedlyCard
+        aspectRatio={config.width / config.height}
+        url={resource?.url}
+        embedlyKey={config.key}
+      />
+    )
+  } else if (resource?.image) {
+    return (
+      <Image
+        src={resourceThumbnailSrc(resource?.image, config)}
+        aspectRatio={config.width / config.height}
+        alt={resource?.image.alt ?? ""}
+      />
+    )
+  } else {
+    return <SkeletonImage variant="rectangular" height={220} />
+  }
+}
+
+const ResourceTitle = ({ resource }: { resource?: LearningResource }) => {
+  if (!resource) {
+    return <Skeleton variant="text" height={20} width="66%" />
+  }
+  return (
+    <Typography variant="h5" component="h2">
+      {resource.title}
+    </Typography>
+  )
+}
+
+const ResourceDescription = ({ resource }: { resource?: LearningResource }) => {
+  if (!resource) {
+    return (
+      <>
+        <Skeleton variant="text" width="100%" />
+        <Skeleton variant="text" width="100%" />
+        <Skeleton variant="text" width="100%" />
+        <Skeleton variant="text" width="100%" />
+        <Skeleton variant="text" width="100%" />
+        <Skeleton variant="text" width="33%" />
+      </>
+    )
+  }
+  return <Description>{resource.description}</Description>
+}
+
+const TopicsSection: React.FC<{ topics: LearningResourceTopic[] }> = ({
+  topics,
+}) => {
+  return (
+    <Topics>
+      <Typography variant="subtitle2" component="h3">
+        Topics
+      </Typography>
+      <TopicsList>
+        {topics.map((topic) =>
+          topic.channel_url ? (
+            <ChipLink
+              key={topic.id}
+              size="medium"
+              label={topic.name}
+              href={topic.channel_url}
+              variant="outlined"
+            />
+          ) : (
+            <Chip
+              key={topic.id}
+              size="medium"
+              label={topic.name}
+              variant="outlined"
+            />
+          ),
+        )}
+      </TopicsList>
+    </Topics>
+  )
+}
+
+const InfoSection = ({ run }: { run?: LearningResourceRun }) => {
+  if (!run) {
+    return null
+  }
+  return (
+    <InfoItems>
+      <Typography variant="subtitle2" component="h3">
+        Info
+      </Typography>
+      <InfoItem>
+        <RiMoneyDollarCircleFill />
+        <InfoLabel>Cost:</InfoLabel>
+        <InfoValue>{run.prices?.[0] || "-"}</InfoValue>
```

Comment:
Prices are returned on the API as an array, e.g. `["0.00", "300.00", "149.00", "300.00"]`. Which to display? Are they always USD? Should we display `"0.00"` as `Free`?

Also, prices are on both the learning resource and child runs. On the parent resource they are numbers (e.g. `[0, 75, 300, 49, 50, 149]`), though strings on the runs `["0.00", "300.00", "149.00", "300.00"]`.